docs/storage.md

```diff
@@ -62,7 +63,7 @@ config:
     enable: false
 ```
 
-AWS region to endpoint mapping can be found in this [link](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region)
+The variable `region` is optional. AWS region to endpoint mapping can be found in this [link](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).
```

Comment:
I am wondering if it would make more sense to have this comment inline with the config something like below and maybe move forward with this approach?
```yaml
config:
  bucket: ""        # required
  endpoint: ""      # required
  region: ""        # optional
  access_key: ""
  insecure: false
  signature_version2: false
```

Reply:
I rollbacked this change. The CI runs each help command to compare with the configuration block (see file `scripts/genflagdocs.sh`).